wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -170,27 +177,29 @@ newAddress
     -> PassPhrase
     -> AccountId
     -> m CAddress
-newAddress addGenSeed passphrase accId =
+newAddress addGenSeed passphrase accId = do
+    balAndUtxo <- WS.getWalletBalancesAndUtxo
     fixCachedAccModifierFor accId $ \accMod -> do
         -- check whether account exists
-        _ <- getAccount accMod accId
+        _ <- getAccountMod (balAndUtxo, accMod) accId
```

Comment:
Yes. We don't need the old style of `getAccount` they can all use this style, and so we can stick to just one `getAccount` action.